src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -441,6 +444,23 @@ impl HummockEventHandler {
                     UploaderEvent::DataSpilled(staging_sstable_info) => {
                         self.handle_data_spilled(staging_sstable_info);
                     }
+
+                    UploaderEvent::ImmMerged(merge_output) => {
```

Comment:
Don't understand the problem here. I already modified the `poll` of `NextUploaderEvent` to emit the `ImmMerged` event.

Reply:
Currently when we handle the `ImmMerged` event, we call `self.uploader.update_sealed_data(...)`. However, the `ImmMerged` is coming out from the uploader. Therefore, we should not handle the `self.uploader.update_sealed_data(...)` in the event handler. Instead, we may handle `self.uploader.update_sealed_data(...)` before emitting the event, because the logic is internal for the uploader.

Reply:
Got it. fixed.